spark/components/modals/react/SprkModal.test.js

```diff
@@ -2,7 +2,6 @@ import React from 'react';
 import Enzyme, { mount, shallow } from 'enzyme';
 import sinon from 'sinon';
 import Adapter from 'enzyme-adapter-react-16';
-import '../windowStubs';
```

Comment:
@RCopeland looks like you removed an import to windowStubs here. Was that intentional?

Reply:
i put it into the setup file directly, no need for a separate file.